salesagility/SuiteCRM
# fixed #6254 - View Change log produces following error in log: [FATAL…

…] SugarBean::populateDefaultValues $field_defs should be an array



## Description
added new condition that excludes the audit module because the audit module doesn't have any var-defs




## Motivation and Context


## How To Test This

1. Go to accounts module
2. Pick view change log action
3. Check SuiteCRM log



## Types of changes


### Final checklist
